src/Controller/CRUDController.php

```diff
@@ -332,13 +333,15 @@ public function editAction($id = null)
         $this->admin->setSubject($existingObject);
         $objectId = $this->admin->getNormalizedIdentifier($existingObject);
 
-        if (!\is_array($fields = $this->admin->getForm()->all()) || 0 === \count($fields)) {
+        /** @var $form Form */
```

Comment:
please use `assert($form instanceof Form)` instead, it has the same effect, but is more powerful (it will break in dev and only there if this is false).

Reply:
This breaks my application as I use a decorator class implementing the FormInterface for a special use case. Using `assert($form instanceof FormInterface)` will fix this issue.

Reply:
@jordykouters could you provide a PR to fix this?

Reply:
Yes, I created a PR in #5355